string type
---
nav_msgs/OccupancyGrid map
